src/test/shell/bazel/workspace_test.sh

```diff
@@ -104,7 +104,7 @@ genrule(
 EOF
   bazel fetch //:test || fail "Fetch failed"
   bazel build //:test || echo "Expected build to succeed"
-  check_eq "12" "$(cat bazel-genfiles/test.out | tr -d '[[:space:]]')"
+  check_eq "12" "$(cat bazel-genfiles/test.out | tr -d '[:space:]')"
```

Comment:
I changed this in accordance to [SC2021](https://github.com/koalaman/shellcheck/wiki/SC2021) but now I wonder about the integrity of the constant (12). And I don't quite understand why it uses double brackets till now.

Reply:
It should be ok if the test passes, both syntax are equivalent.

Reply:
Well, the first one is supposed to treat the outer brackets as literals.

Reply:
Humm it certainly does not on my computer:

```
[dmarting@dmarting-macbookpro git bazel (source_date_epoch)] $ echo 'a b c' | tr -d '[:space:]'; echo
abc
[dmarting@dmarting-macbookpro git bazel (source_date_epoch)] $ echo 'a b c' | tr -d '[[:space:]]'; echo
abc
```

Reply:
`echo 'a[b c]d' | tr -d '[:space:]'` produces `a[bc]d`
`echo 'a[b c]d' | tr -d '[[:space:]]'` produces `abcd`

Reply:
Oh I see, there is no way this file contains bracket so we are fine with that change.

Reply:
(also seeing the code, it is pretty clear the intention was to only remove whitespaces here)

Reply:
What does the constant (12) refers to?

Reply:
It's 1 and 2 concatenated, it correspond to the two number written by the genrules